app/views/locations/_form.html.erb

```diff
@@ -49,6 +49,18 @@
               <%= f.label :Country , class: 'form-label' %><br>
               <%= f.text_field :country %>
             </div>
+            <div class="mb-3">
+              <%= f.label :'Start Date' , class: 'form-label' %><br>
+              <%= f.text_field :start_date, type: 'date', value: @location.start_date &.to_date %>
+            </div>
+            <div class="mb-3">
+              <%= f.label :'End Date' , class: 'form-label' %><br>
+              <%= f.text_field :end_date, type: 'date', value: @location.end_date &.to_date %>
+            </div>
+            <div class="mb-3">
+              <%= f.label :'Exclude Dates' , class: 'form-label' %><br>
+              <%= f.text_field :exclude_dates %>
```

Comment:
This should be changed to a multi-select menu, listing the available dates in the program year.

https://trello.com/c/A02pUA3R/701-update-locations-form-to-multi-select-for-excluded-dates